web/src/search/input/SearchNavbarItem.tsx

```diff
@@ -20,52 +36,108 @@ interface Props
         SettingsCascadeProps,
         ThemeProps,
         CopyQueryButtonProps,
-        VersionContextProps {
+        VersionContextProps,
+        OnboardingTourProps {
     location: H.Location
     history: H.History
     navbarSearchState: QueryState
     onChange: (newValue: QueryState) => void
     globbing: boolean
 }
 
+function createStructuralSearchTourTooltip(): HTMLElement {
+    const listItem = document.createElement('li')
+    listItem.className = 'list-group-item p-0 border-0 my-4'
+    listItem.textContent = '>'
```

Comment:
```suggestion
    listItem.textContent = '>'
```

Can we remove this line? It was a poor inclusion on my part.

Reply:
This results in the > before the documentation link.

Reply:
+1, the `li` should be inside a `ul` here

Reply:
Will do after merging this into the other branch so I don't have to do it on two different branches.